### ISSUE TITLE
Accepting a contact request corrupts chat list

### ISSUE THREAD

**Issue by adbenitez** [deltachat/deltachat-desktop#1283]
When I accept a contact request all the chats from the chat list disappear and only the accepted chat is shown in the list.

- Platform (linux/mac/windows): GNU/Linux
- Delta Chat Desktop Version: master

**Comment by adbenitez** [comment]
notice this used to work some days ago

**Comment by Jikstra** [comment]
Also clicking on not now corrupts the chatlist. Here's the log:
```
logger.js:56 🕸️renderer/ipc backend DD_EVENT_CHATLIST_CHANGED {}
logger.js:56 🕸️renderer/helpers/ChatList useChatListIds: refetchingChatlist
logger.js:56 🕸️renderer/ipc backend DD_EVENT_CHATLIST_ITEM_CHANGED {chatId: 0}
logger.js:56 🕸️renderer/ipc sendToBackend: EVENT_DC_DISPATCH_CB 272 chatList.getChatListItemsByIds 1,12,13,10,18,11
logger.js:56 🕸️renderer/ipc backend DD_EVENT_CHAT_MODIFIED {chat: null, chatId: 0}
logger.js:56 🕸️renderer/ipc EVENT_DD_DISPATCH_RETURN_272_chatList.getChatListItemsByIds Got back return:  {1: {…}, 10: {…}, 11: {…}, 12: {…}, 13: {…}, 18: {…}}
logger.js:56 🕸️renderer/helpers/ChatList useLazyChatListItems: Force updating chats in view, unsetting others (6) ["1", "10", "11", "12", "13", "18"]
logger.js:56 🕸️renderer/helpers/ChatList useLazyChatListItems: chatListIds changed, updating chats in view
logger.js:56 🕸️renderer/ipc sendToBackend: EVENT_DC_DISPATCH_CB 273 chatList.getChatListIds 0,,
logger.js:56 🕸️renderer/ipc EVENT_DD_DISPATCH_RETURN_273_chatList.getChatListIds Got back return:  (6) [1, 12, 13, 10, 18, 11]
logger.js:56 🕸️renderer/helpers/ChatList useLazyChatListItems: chatListIds changed, updating chats in view
logger.js:56 🕸️renderer/dialogs openDialog:  DeadDrop {contact: {…}, filemime: "", filename: "", filesize: 0, fromId: 20, …}
logger.js:56 🕸️renderer/ipc sendToBackend: EVENT_DC_DISPATCH_CB 274 contacts.acceptContactRequest [object Object]
logger.js:56 🕸️renderer/ipc backend DD_EVENT_CHATLIST_UPDATED null
logger.js:56 🕸️renderer/ipc EVENT_DD_DISPATCH_RETURN_274_contacts.acceptContactRequest Got back return:  15
logger.js:56 🕸️renderer/stores/ChatStore DISPATCH: {type: "SELECT_CHAT", payload: 15}
logger.js:56 🕸️renderer/ipc sendToBackend: EVENT_DC_DISPATCH_CB 275 chatList.selectChat 15
logger.js:56 🕸️renderer/ipc backend DC_EVENT_CONTACTS_CHANGED (2) [0, 0]
logger.js:56 🕸️renderer/ipc backend DC_EVENT_MSGS_CHANGED (2) [0, 0]
logger.js:56 🕸️renderer/stores/ChatStore DC_EVENT_MSGS_CHANGED 0 0
logger.js:56 🕸️renderer/ipc backend DD_EVENT_CHATLIST_CHANGED {}
logger.js:56 🕸️renderer/helpers/ChatList useChatListIds: refetchingChatlist
logger.js:56 🕸️renderer/ipc backend DD_EVENT_CHATLIST_ITEM_CHANGED {chatId: 0}
logger.js:56 🕸️renderer/ipc sendToBackend: EVENT_DC_DISPATCH_CB 276 chatList.getChatListItemsByIds 1,12,13,10,18,11
logger.js:56 🕸️renderer/ipc backend DD_EVENT_CHAT_MODIFIED {chat: null, chatId: 0}
logger.js:56 🕸️renderer/ipc EVENT_DD_DISPATCH_RETURN_275_chatList.selectChat Got back return:  {archived: 0, color: "#76c84d", contactIds: Array(1), contacts: Array(1), draft: "", …}
logger.js:56 🕸️renderer/ipc sendToBackend: EVENT_DC_DISPATCH_CB 277 messageList.getMessageIds 15
logger.js:56 🕸️renderer/ipc backend DC_EVENT_MSGS_CHANGED (2) [0, 0]
logger.js:56 🕸️renderer/stores/ChatStore DC_EVENT_MSGS_CHANGED 0 0
logger.js:56 🕸️renderer/ipc backend DD_EVENT_CHATLIST_CHANGED {}
logger.js:56 🕸️renderer/helpers/ChatList useChatListIds: refetchingChatlist
logger.js:56 🕸️renderer/ipc backend DD_EVENT_CHATLIST_ITEM_CHANGED {chatId: 0}
logger.js:56 🕸️renderer/ipc backend DD_EVENT_CHAT_MODIFIED {chat: null, chatId: 0}
logger.js:56 🕸️renderer/ipc backend DC_EVENT_MSGS_CHANGED (2) [0, 0]
logger.js:56 🕸️renderer/stores/ChatStore DC_EVENT_MSGS_CHANGED 0 0
logger.js:56 🕸️renderer/ipc backend DD_EVENT_CHATLIST_CHANGED {}
logger.js:56 🕸️renderer/helpers/ChatList useChatListIds: refetchingChatlist
logger.js:56 🕸️renderer/ipc backend DD_EVENT_CHATLIST_ITEM_CHANGED {chatId: 0}
logger.js:56 🕸️renderer/ipc backend DD_EVENT_CHAT_MODIFIED {chat: null, chatId: 0}
logger.js:56 🕸️renderer/ipc EVENT_DD_DISPATCH_RETURN_276_chatList.getChatListItemsByIds Got back return:  {1: {…}, 10: {…}, 11: {…}, 12: {…}, 13: {…}, 18: {…}}
logger.js:56 🕸️renderer/helpers/ChatList useLazyChatListItems: Force updating chats in view, unsetting others (6) ["1", "10", "11", "12", "13", "18"]
logger.js:56 🕸️renderer/ipc EVENT_DD_DISPATCH_RETURN_277_messageList.getMessageIds Got back return:  (2) [9, 76]
logger.js:56 🕸️renderer/ipc sendToBackend: EVENT_DC_DISPATCH_CB 278 messageList.getMessages 9,76
logger.js:56 🕸️renderer/helpers/ChatList useLazyChatListItems: chatListIds changed, updating chats in view
logger.js:56 🕸️renderer/ipc EVENT_DD_DISPATCH_RETURN_278_messageList.getMessages Got back return:  {76: {…}}
```

**Comment by pabzm** [comment]
This appears to be fixed in  #1382, can't reproduce it anymore with the latest master.

@adbenitez If it's not resolved for you please re-open!
